backend/migrations/versions/a5d22b31faab_add_resident_table.py

```diff
@@ -25,8 +25,10 @@ def upgrade():
         sa.Column("room_num", sa.Integer(), nullable=False),
         sa.Column("date_joined", sa.DateTime(timezone=True), nullable=False),
         sa.Column("date_left", sa.DateTime(timezone=True), nullable=True),
-        sa.Column(
-            "building", sa.Enum("144", "402", "362", name="buildings"), nullable=False
+        sa.Column("building_id", sa.Integer(), nullable=False),
+        sa.ForeignKeyConstraint(
+            ["building_id"],
+            ["buildings.id"],
```

Comment:
were you able to run `flask db migrate` after your changed the `Resident` model? you shouldn't need to edit migration files once they are created

basically how it works is everytime you make a model change a new migration file is created (never edited), so ideally in this case, there should be a new migration file that updates this column

Reply:
uhhhh i'm sure i was able to flask db migrate

Reply:
okay nw then it should be fine but next time you have to update modelsjust run `flask db migrate` and it will do it all of this for you